configs/twins/twins_alt-gvt-b_uperhead_8x2_512x512_160k_ade20k.py

```diff
@@ -0,0 +1,13 @@
+_base_ = ['twins_alt-gvt-s_uperhead_8x2_512x512_160k_ade20k.py']
```

Comment:
alt-gvt  or altgvt?
Should use consistent name, s or small (full name or shot name).